Saas/New-BcEnvironment.ps1

```diff
@@ -20,6 +20,8 @@
   The version of the application the environment should be created on
  .Parameter applicationInsightsKey
   Application Insights Key to add to the environment
+ .Parameter apiVersion
```

Comment:
apiVersion is mentioned here - but it isn't added as a parameter (i.e. would be undefined in the code)

Reply:
Done